carsus/io/output/base.py

```diff
@@ -486,42 +544,55 @@ def _create_levels_lines(self, lines_loggf_threshold=-3,
         levels = levels.sort_values(
             ["atomic_number", "ion_number", "level_number"])
 
-        self.lines = lines
-        self.levels = levels
+        return levels, lines
 
-    def _create_collisions(self):
-        # Exclude artificially created levels from levels
-        levels = self.levels.loc[self.levels["level_id"] != -1].set_index("level_id")
+    def create_collisions(self, temperatures=np.arange(2000, 50000, 2000)):
```

Comment:
is that a sensible temperature range? This should be somewhere better documented.

Reply:
I don't know. It was used in legacy code.

Reply:
I think this would be worth putting in the documentation.